sdk/src/connections/target/adsd3500_sensor.cpp

```diff
@@ -170,26 +171,25 @@ Adsd3500Sensor::~Adsd3500Sensor() {
         for (unsigned int i = 0; i < dev->nVideoBuffers; i++) {
             if (munmap(dev->videoBuffers[i].start,
                        dev->videoBuffers[i].length) == -1) {
-                LOG(WARNING)
-                    << "munmap error "
-                    << "errno: " << errno << " error: " << strerror(errno);
+                LOG(WARNING) << "munmap error " << "errno: " << errno
```

Comment:
There are a lot of changes related to code formatting. Why?

Reply:
Those changes were made automatically by vscode option "format on save". I will remove them.